OpenRA.Game/TextNotificationsManager.cs

```diff
@@ -21,6 +21,7 @@ public static class TextNotificationsManager
 		static readonly string SystemMessageLabel;
 
 		public static long ChatDisabledUntil { get; internal set; }
+		public static int? KickVotePlayer { get; internal set; }
```

Comment:
Imo conceptual it doesn't belong here.

Reply:
sure, but where should I put it?

Reply:
I could place it inside `OrderManager`, or I could add it to `UnitOrders` and then add a `Clear()` to be just like `TextNotificationsManager`

Reply:
moved to `UnitOrder`, the class where it's set